backend/handler/class.go

```diff
@@ -83,3 +83,50 @@ func CreateInvite(cc echo.Context) error {
 		"inviteCode": inviteCode,
 	})
 }
+
+// Functionality : add User to a class Endpoint
+// step1 : verify if classId exists
+// step2 : get the user type
+// step3 : add user to ClassMembers table
+// No body content required as of now
+func EnrollStudent(cc echo.Context) error {
```

Comment:
You're on the right track with this, but since our system is built on invites, I think we can get away with something like this:

```mermaid
flowchart TD
    Z[Make request containing an invite code] --> A{Invite code exists}
    A -->|Yes| C[Retrieve class code associated to invite code]
    C --> E[Insert user to class]
    A -->|No| D[Return 4xx]
```

We can then retrieve the associated class code from our existing schema:

```sql
SELECT invites_to
FROM Invites
WHERE id = $1
  AND expires >= CURRENT_TIMESTAMP
```

If this returns a row, then we know the invite was valid!